druid-shell/src/platform/gtk/window.rs

```diff
@@ -332,10 +333,11 @@ impl WindowBuilder {
                         handler.wheel(Vec2::from((120.0, 0.0)), modifiers, &mut ctx);
                     }
                     ScrollDirection::Smooth => {
-                        // TODO: support smooth scrolling via scroll.get_delta and get_is_stop
-                        eprintln!(
-                            "Warning: somehow the Druid widget got a smooth scroll event"
-                        );
+                        //TODO: Look at how gtk's scroll containers implements it
+                        let (mut delta_x, mut delta_y) = scroll.get_delta();
+                        delta_x *= 120.;
```

Comment:
I appreciate this is sort of unrelated to this patch, but it might be a good time to break out this `120.0` into some const somewhere with an informative name? Do we mention anywhere how we chose this value?

Also you can multiply a `Vec2` by a float, so you should be able to just write,

```rust
let scroll_delta = scroll.get_delta() * SCROLL_SCALE_FACTOR;
```
or something similar.

Reply:
There is a comment about it a few lines up
"The magic "120"s are from Microsoft's documentation for WM_MOUSEWHEEL. They claim that one "tick" on a scroll wheel should be 120 units."

It should probably be refactored into a const, I'll get on it when i have the time.

I am also interested in looking at how gnome handles scrolling in their widgets and mabye try to implement some form of kinetic scroll, but that will require some rework of the scroll event sent to druid.

Reply:
sounds good! I have some minor concerns about gtk borrowing this from a windows implementation detail but it's probably a reasonable default. That said, it might be worth digging into that, as well...